Jen_Surovy.rb

jm2208 opened the pull request.

acodeinprogress commented:
Hi @jm2208, looks good! I do have one request for you before I can merge this in. We would like for your Ruby file to be in the Ruby folder. If you could go ahead and move it, then run git add -A, commit, and push, your changes will join your pull request here. Then I can get everything merged in!
Let me know if you need any help!

jm2208 commented:
So I got over eager and deleted the file (instead of following your instructions :() I added the doc on my computer, and tried to request a new pull, but takes me back here.  What steps should I take from this point? Sorry for the problems here!

acodeinprogress commented:
Hey @jm2208,

That's no problem at all! All you have to do is add, commit, and push. All of your changes will automatically join this pull request!

Let me know if you need help!
